doc/sphinx/source/projects/ad405x_iio/ad405x_iio.rst

```diff
@@ -0,0 +1,96 @@
+AD405x IIO Application
+""""""""""""""""""""""
+
+==================
+Supported Hardware
+==================
+
+**Supported Devices:**
+
+* `AD4052 <https://www.analog.com/en/products/ad4052.html>`_ 
+
+**Supported Evaluation Boards:**
+
+* `EVAL-AD4052-ARDZ <https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad4052.html>`_
+
+**Supported Carrier Boards:**
+
+* `SDP-K1 With Mbed Platform <https://os.mbed.com/platforms/SDP_K1/>`_
+
+============
+Introduction
+============
+
+This page gives an overview of using the ARM platforms supported (default is Mbed) 
+firmware example with Analog Devices AD4052 Evaluation board and SDP-K1 controller board. 
+This example code leverages the ADI developed IIO (Industrial Input Output) ecosystem to 
+evaluate the AD4052 device by providing a device debug and data capture support.
+The code provides support for MBED and STM32 platforms.
```

Comment:
Would it be a good idea to mention that SDP-K1 is the MCU board for both mbed and STM32?